cmd/server/main.go

```diff
@@ -1,3 +1,37 @@
 package main
 
-func main() {}
+import (
+	"flag"
+	"log"
+	"net/http"
+
+	"github.com/andrewsvn/metrics-overseer/internal/config/servercfg"
+	"github.com/andrewsvn/metrics-overseer/internal/handler"
+	"github.com/andrewsvn/metrics-overseer/internal/repository"
+	"github.com/andrewsvn/metrics-overseer/internal/service"
+)
+
+func main() {
+	log.Fatal(run())
+}
+
+func run() error {
+	cfg := readConfig()
+
+	mstor := repository.NewMemStorage()
+	msrv := service.NewMetricsService(mstor)
+	mhandlers := handler.NewMetricsHandlers(msrv)
+
+	r := mhandlers.GetRouter()
+
+	log.Printf("Starting server on %s\n", cfg.Addr)
```

Comment:
Когда пользуешься пакетом `log` можно не добавлять новую строку
```suggestion
	log.Printf("Starting server on %s", cfg.Addr)
```